packages/theme/stories/introduction.stories.mdx

```diff
@@ -106,6 +107,23 @@ The full set of characteristics is currently defined is as follows:
 | Taggable            | components which allow data to be organized                                       | `--uitk-taggable-background: var(--uitk-palette-interact-primary-background);`                        |
 | Text                | all text components or HTML elements                                              | `--uitk-text-primary-foreground: var(--uitk-palette-neutral-primary-foreground);`                     |
 
+And finally, the palette:
+
+| Palette sentiment | Appropriate usage                                               | Example token                                            |
+| ----------------- | --------------------------------------------------------------- | -------------------------------------------------------- |
+| Accent            | characteristics which use the main brand color                  | `--uitk-palette-accent-background`                       |
```

Comment:
Just FYI, [`accent-color`](https://developer.mozilla.org/en-US/docs/Web/CSS/accent-color) is in CSS and like Mac OS level setting, not sure whether us associate it to "brand" would cause future issue